template/extra_sprig.go

```diff
@@ -62,15 +81,19 @@ var sprigFuncRef = map[string]struct {
 	group, description string
 }{
 	"hello": {group: sprigGen, description: "Simple hello by Sprig"},
+
 	// Date functions
 	"date":           {group: sprigDate, description: "The date function formats a dat (https://golang.org/pkg/time/#Time.Format).", arguments: []string{"fmt", "date"}},
 	"now":            {group: sprigDate, description: "The current date/time. Use this in conjunction with other date functions."},
 	"htmlDate":       {group: sprigDate, description: "The htmlDate function formates a date for inserting into an HTML date picker input field.", arguments: []string{"date"}},
 	"htmlDateInZone": {group: sprigDate, description: "Same as htmlDate, but with a timezone.", arguments: []string{"date", "zone"}},
 	"dateInZone":     {group: sprigDate, description: "Same as date, but with a timezone.", arguments: []string{"fmt", "date", "zone"}, aliases: []string{"date_in_zone"}},
-	"dateModify":     {group: sprigDate, description: "The dateModify takes a modification and a date and returns the timestamp.", arguments: []string{"fmt", "date"}, aliases: []string{"date_modify"}},
+	"dateModify":     {group: sprigDate, description: "The dateModify takes a modification and a date and returns the timestamp.", arguments: []string{"fmt", "date"}},
+	"date_modify":    {group: sprigDate, description: "The dateModify takes a modification and a date and returns the timestamp.", arguments: []string{"fmt", "date"}},
```

Comment:
Why add this function? The different casing just seems weird

Reply:
It comes from legacy Sprig. I just moved it from an alias to a function since they add `must_date_modify`.